src/Components/RelatedProducts/RPComponents/OutfitCard/OutfitCard.jsx

```diff
@@ -1,12 +1,19 @@
+/* eslint-disable react/destructuring-assignment */
 import React from 'react';
 import './OutfitCard.scss';
 
 const OutfitCard = (props) => (
-  <button type="button" className="add-item" onClick={() => props.handleClick}>
+  <div
```

Comment:
Add 'rp-' Prefix to Privatize CSS Classes
Expanded for Readability, Code CleanUp